CHANGELOG.md

```diff
@@ -3,8 +3,8 @@
 All notable changes to the Zowe CLI package will be documented in this file.
 
 ## Recent Changes
-- Enhancement: Added options like `--lrecl, --recfm` to zos-files create options. [788](https://github.com/zowe/zowe-cli/issues/788)
-***
+
+- Added `--pattern` option to `zowe files list am`
```

Comment:
Please add a label (e.g. `Enhancement:`) and the issue number to the changelog.
Just trying to follow what the Zowe Docs Squad suggested 😋

Reply:
Can you link the doc squad comments? I don't think this makes sense and conflicts with our guidelines in our repo.